edx-sandbox/tests/test_edx.sls

```diff
@@ -0,0 +1,30 @@
+{% from "edx-sandbox/map.jinja" import edx_sandbox with context %}
+
+# Do some simple filesystem checks that will certainly fail if edX wasn't deployed.
+
+{%
+set fs_validation = {
+  'edxapp_env': {'type': 'file', 'loc': '/edx/app/edxapp/edxapp_env'},
+  'nginx_sites': {'type': 'directory', 'loc': '/edx/app/nginx/sites-available'},
+  'supervisor_bin': {'type': 'file', 'loc': '/edx/bin/supervisorctl'},
+  'edxapp_manage_bin': {'type': 'symlink', 'loc': '/edx/bin/manage.edxapp', 'link': '/edx/app/edxapp/edx-platform/manage.py'},
+  'edxapp_pip_bin': {'type': 'symlink', 'loc': '/edx/bin/pip.edxapp', 'link': '/edx/app/edxapp/venvs/edxapp/bin/pip'},
+}
+%}
+
+{% for name, props in fs_validation.items() %}
+test_{{ name }}_file_exists:
+  testinfra.file:
+    - name: {{ props.loc }}
+    - is_{{ props['type'] }}: True
```

Comment:
You're mixing access patterns here. Just a nit, but might want to just use either . notation of [] notation everywhere :)